src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.cs

```diff
@@ -36,7 +36,7 @@ public async Task LangVersionMustBeCharp12OrHigher(LanguageVersion langVersion)
             Assert.Equal(DiagnosticSeverity.Error, diagnostic.Severity);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNetCore))]
```

Comment:
>IsNetCore [](http://example.com/codeflow?start=77&length=9)

Why changing this to be Net Core only?

Reply:
In main branch, this test already executes only in .NETCore so I'm not changing anything.


https://github.com/dotnet/runtime/blob/216dd61512311698dd321ce1c4880f9dad14366a/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Microsoft.Extensions.Configuration.Binder.SourceGeneration.Tests.csproj#L51C3-L57C44

For the reason on why it is excluded while other tests are now getting executed in net462 : it uses the record feature.

Reply:
should we move the netcore specific baseline files to a folder like NetCoreApp, and we do the similar idea for NETFX too.

Reply:
not sure if I'm understanding your remark well : We already have different baseline files depending on whether they are in `Baselines\netcoreapp` or `Baselines\net462`.